nvflare/private/aux_runner.py

```diff
@@ -141,6 +143,117 @@ def dispatch(self, topic: str, request: Shareable, fl_ctx: FLContext) -> Shareab
                 valid_reply.set_cookie_jar(cookie_jar)
         return valid_reply
 
+    def _wait_for_cell(self):
+        start = time.time()
+        self.logger.debug("waiting for cell...")
+        max_wait = 5.0
```

Comment:
This was refactored from previous code (put into this reusable func since it will be used in multiple places now). But I think you are right - we should make it part of the config.